src/pages/AdditionalInformation/PrivacyNotices.jsx

```diff
@@ -17,15 +17,15 @@ const TermsOfUse = () => {
             >
               Često postavljana pitanja
             </NavLink>
-            <br />
+            <br /> <br />
```

Comment:
Use some different approach

Reply:
Deleted

Reply:
Changes are found on the fixed impressions component